src/sentry/dynamic_sampling/rules/biases/boost_key_transactions_bias.py

```diff
@@ -59,8 +36,8 @@ def _generate_bias_rules(self, bias_data: BiasData) -> List[PolymorphicRule]:
         return [
             {
                 "samplingValue": {
-                    "type": "sampleRate",
-                    "value": bias_data["sampleRate"],
+                    "type": "factor",
+                    "value": bias_data["factor"],
```

Comment:
Could we split the cleanup (removal of V1 generators) and the functional change into separate PRs? Should be a minor effort but will make it easier to review the functional change, both now and when referencing the PR in the future.

Reply:
Sure, I will create a separate PR with the implementation of dynamic factor and this one will contain the removal of v1.